### ISSUE TITLE
A word of caution.

### ISSUE THREAD

**Issue by Ariakim-Taiyo** [NVB9ALT/Helment-Mounted-Displays#1]
Xavier has not taken kindly to attempts to send extra metadata through the multiplayer servers in the past. The SkyX addon was shut down because of that. 
My addons only use existing data to communicate on multiplayer, for example the multiplayer livery visibility in my livery addons relies on an existing property used for that purpose, only instead of using default values I use values that do not correspond to a default geofs livery.

Keep that in mind if you decide to proceed, as technically you are not allowed to send extra data through GeoFS's servers.

**Comment by NVB9ALT** [comment]
And technically, I'm not. Originally I added a new variable to the list in var f, "is: isShooting", but that actually didn't work. So instead, I made gr turn on when the player was shooting someone down instead of when they were in contact with the ground, which did work. The exact same effect could be achieved by setting geofs.animation.values.groundContact to the value of var shooting, so as far as Xavier knows, aircraft are just being in contact with the ground at strange times.
